addons/pharma/functions/fnc_treatmentAdvanced_TXALocal.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+/*
+ * Author: 2LT.Mazinski
+ * Begins TXA bandaging process
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Medication <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "TXA"] call kat_circulation_fnc_treatmentAdvanced_TXALocal;
+ *
+ * Public: No
+ */
+
+params ["_target", "_item"];
+
+[_target, _item] call ace_medical_treatment_fnc_addToTriageCard;
+[_target, "activity", LSTRING(push_log), [[_medic] call ace_common_fnc_getName, "TXA"]] call ace_medical_treatment_fnc_addToLog;
+[_target, "TXA", 5, 120, 0, 0, 0] call ace_medical_status_fnc_addMedicationAdjustment;
+
+
+[{
+    params ["_args", "_idPFH"];
+    _args params ["_target"];
+
+    private _medicationArray = _target getVariable ["ace_medical_medications", []];
+    private _TXA = false;
+    private _TXAactive = true;
+
+    {
+        _x params ["_medication"];
+
+        if (_medication == "TXA") exitWith {
+            _TXA = true;
+       };
+    } forEach (_medicationArray);
+
+    private _alive = alive _target;
+
+    if ((!_alive) || (_TXA == false)) exitWith {
+        [_idPFH] call CBA_fnc_removePerFrameHandler; 
```

Comment:
ALso add here this behavior for the pfh `_unit setVariable ["kat_TXA_PFH", nil];`

Reply:
Check that I did this right please.